dashboard/test/ui/features/teacher_tools/rubrics/ai_evaluate_student_code.feature

```diff
@@ -87,7 +86,9 @@ Feature: Evaluate student code against rubrics using AI
     And element ".teacher-panel td:eq(1)" contains text "Aiden"
     And I click selector ".teacher-panel td:eq(1)" to load a new page
     And I wait for the page to fully load
+    And I wait until element "h1:contains(Getting Started with Your AI Teaching Assistant)" is visible
     And I click selector ".introjs-skipbutton" if it exists
+    And I wait for 2 seconds
```

Comment:
waiting for a number of seconds is a great debugging strategy, but can unfortunately be a source of flakiness. Since you are clicking the skip button, can you wait until the intro dialog is gone instead? for example: https://github.com/code-dot-org/code-dot-org/blob/826f1875eb05910a380c77cdaa448f64a1c27872/dashboard/test/ui/features/star_labs/craft/dialogs.feature#L33